src/User_Command.php

```diff
@@ -692,26 +692,38 @@ public function set_role( $args, $assoc_args ) {
 	 * <user>
 	 * : User ID, user email, or user login.
 	 *
-	 * <role>
-	 * : Add the specified role to the user.
+	 * [<role>...]
+	 * : Add the specified role(s) to the user.
 	 *
 	 * ## EXAMPLES
 	 *
 	 *     $ wp user add-role 12 author
 	 *     Success: Added 'author' role for johndoe (12).
 	 *
+	 *     $ wp user add-role 12 author editor
+	 *     Success: Added 'author' role for johndoe (12).
+	 *     Success: Added 'editor' role for johndoe (12).
+	 *
 	 * @subcommand add-role
 	 */
 	public function add_role( $args, $assoc_args ) {
 		$user = $this->fetcher->get_check( $args[0] );
 
-		$role = $args[1];
+		$roles = $args;
+		array_shift( $roles );
 
-		self::validate_role( $role );
+		if ( empty( $roles ) ) {
+			WP_CLI::error( 'Please specify at least one role to add.' );
+		}
 
-		$user->add_role( $role );
+		foreach ( $roles as $role ) {
+			self::validate_role( $role );
+		}
 
-		WP_CLI::success( "Added '{$role}' role for {$user->user_login} ({$user->ID})." );
+		foreach ( $roles as $role ) {
+			$user->add_role( $role );
+			WP_CLI::success( "Added '{$role}' role for {$user->user_login} ({$user->ID})." );
```

Comment:
Rather than render `Success:` multiple times, can we render it once and mention both roles?